src/assets/Icon/Icon.jsx

```diff
@@ -0,0 +1,55 @@
+
+export function MdiHomeOutline(props) {
+    return (
+      <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}><path fill="currentColor" d="m12 5.69l5 4.5V18h-2v-6H9v6H7v-7.81l5-4.5M12 3L2 12h3v8h6v-6h2v6h6v-8h3"></path></svg>
+    )
+  }
+
+
+export function MdiLabelOutline(props) {
+    return (
+      <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}><path fill="currentColor" d="M16 17H5V7h11l3.55 5m-1.92-6.16C17.27 5.33 16.67 5 16 5H5a2 2 0 0 0-2 2v10a2 2 0 0 0 2 2h11c.67 0 1.27-.34 1.63-.85L22 12l-4.37-6.16Z"></path></svg>
+    )
+  }
+
+
```

Comment:
Instead of writing separate component fo the icons
You can use `import {reactComponent as IconName} from "../path/Icon.svg`